tree/tree/Controller/Scrap/ScrapTabBarController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ScrapTabBarController.swift
+//  tree
+//
+//  Created by Hyeontae on 04/02/2019.
+//  Copyright © 2019 gardener. All rights reserved.
+//
+
+import UIKit
+
+class ScrapTabBarController: UITabBarController {
+
+    private var bounceAnimation: CAKeyframeAnimation = {
+        let bounceAnimation = CAKeyframeAnimation(keyPath: "transform.scale")
+        bounceAnimation.values = [1.0, 1.2, 0.8, 1.2, 1.0]
+        bounceAnimation.duration = 0.3
+        bounceAnimation.calculationMode = CAAnimationCalculationMode.cubic
+        return bounceAnimation
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate,
+            let tabBarViewControllers = self.viewControllers else { return }
+        let navigationControllerArr = tabBarViewControllers.filter{ $0 is ScrapConnected }
```

Comment:
[컨벤션]
정해진 컨벤션과 맞지 않는 띄어쓰기입니다.
```suggestion
        let navigationControllerArr = tabBarViewControllers.filter { $0 is ScrapConnected }
```